apps/data/src/pipelines/centrum_pipeline.py

```
@@ -782,6 +784,113 @@ def execute_macro_udf(pdf: pd.DataFrame) -> pd.DataFrame:
 
 # COMMAND ----------
 
+# DBTITLE 1,Gold Layer - Experiment Macro Data (Sandbox Comparison)
+EXPERIMENT_SANDBOX_MACRO_UDF = make_execute_macro_udf(ENVIRONMENT, dbutils)
```

Ruff (0.15.10)

[error] Undefined name `dbutils`

(F821)